Scripts/Core/EditorVR.Interfaces.cs

```diff
@@ -282,6 +293,27 @@ internal void ConnectInterfaces(object obj, Transform rayOrigin = null)
 				if (requestStencilRef != null)
 					requestStencilRef.requestStencilRef = RequestStencilRef;
 
+				var linkedTool = obj as ILinkedTool;
+				if (linkedTool != null)
+				{
+					List<ILinkedTool> otherTools;
+					var type = obj.GetType();
+					if (m_LinkedTools.TryGetValue(type, out otherTools))
+					{
+						foreach (var otherTool in otherTools)
+						{
+							otherTool.otherTools.Add(linkedTool);
+						}
+
+						linkedTool.otherTools.AddRange(otherTools);
+					}
+					else
+					{
+						linkedTool.primary = true;
```

Comment:
The primary tool is just the first one which is created. This is needed in order to do "cross tool" procedures only once per frame, rather than once per frame per tool. We could rename it to "first" or "master" if that helps.